index.js

```diff
@@ -0,0 +1,2 @@
+const nombreusuario = prompt('Ingresa tu nombre');
+alert(`Hola ${nombreusuario}!!`);
```

Comment:
En este caso, me parece major no usar interpolación de string.

```suggestion
alert('Hola ' + nombreusuario);
```